ApplicationLibCode/ModelVisualization/Intersections/RivExtrudedCurveIntersectionGeometryGenerator.cpp

```diff
@@ -206,12 +206,16 @@ void RivExtrudedCurveIntersectionGeometryGenerator::calculateSurfaceIntersection
                 bool foundMatch = RigSurfaceResampler::resamplePoint( surface, pointAbove, pointBelow, intersectionPoint );
                 if ( !foundMatch )
```

Comment:
Remove the line creating a point with z=infinity(), as this point is never used

Reply:
Fixed!